scripts/scil_bundle_score_same_bundle_many_segmentations.py

```diff
@@ -217,9 +219,8 @@ def main():
     else:
         gs_binary_3d = get_data_as_mask(nib.load(args.voxels_measures[0]))
         gs_binary_3d[gs_binary_3d > 0] = 1
-        tracking_mask_data = get_data_as_mask(nib.load(
-                                                args.voxels_measures[1]))
-        tracking_mask_data[tracking_mask_data > 0] = 1
```

Comment:
why did you remove this ? useless ?

Reply:
The get_data_as_mask function now covers for that.

Reply:
The get_data_as_mask function now covers for that.